explainer-server/public/stylesheets/scss/layout/_toolbar.scss

```diff
@@ -151,7 +151,7 @@ $toolbarHeight: 50px;
   height: 100%;
 }
 
-.save-state {
+.publication-state .save-state {
```

Comment:
These will all need a comma after them, otherwise you're just targeting the `.save-state` inside the `.publication-state` - Ignore me if that's what you intended.